README.md

```diff
@@ -70,17 +76,17 @@ Recovering if the sync is not happening with basic pull commands:
 - manually editing the parent variable in noir/noir-repo/.gitrepo: this is the parent of the last sync commit on aztec side. If you get errors with a commit not being found in the upstream repo, and the commit mentioned is not the commit variable above, it might indicate this is somehow incorrect. This can happen when commit content is ported without its history, e.g. squashes
 - use pull --force ONLY where you would use git reset. That is, if you really want to match some upstream noir for a purpose its fine, but you'll lose local changes (if any)
 
-
 ## Earthly
 
 Earthly is a reproducible build tool that aims to combine the functionality of Docker, Makefiles and BASH.
 Non-build earthly targets should start with 'test', 'run', or 'bench' as a general rule (but not hard rule) while builds can be nouns or start with build-.
 If something is a bundle of targets for CI, we can do e.g. build-ci, test-ci etc.
 See barretenberg/cpp/Earthfile for an example of a fairly involved Earthfile that can be used for inspiration.
-Earthly docs https://docs.earthly.dev/ are extensive and show the various build patterns. 
+Earthly docs <https://docs.earthly.dev/> are extensive and show the various build patterns.
```

Comment:
should this be in brackets? or a link?
```suggestion
[Earthly docs](https://docs.earthly.dev/) are extensive and show the various build patterns.
```

Reply:
ofc, don't know why this changed in the first place...